contracts/colonyNetwork/ColonyNetwork.sol

```diff
@@ -187,20 +220,25 @@ contract ColonyNetwork is ColonyDataTypes, BasicMetaTransaction, ColonyNetworkSt
     decayRate.nextNumerator = _numerator;
     decayRate.nextDenominator = _denominator;
 
-    emit ColonyReputationDecayRateToChange(msgSender(), activeReputationMiningCycle, _numerator, _denominator);
+    emit ColonyReputationDecayRateToChange(_chainId, _colony, activeReputationMiningCycle, _numerator, _denominator);
   }
 
-  function getColonyReputationDecayRate(address _colony) public view returns (uint256, uint256) {
+  function getColonyReputationDecayRate(uint256 _chainId, address _colony) public view returns (uint256, uint256) {
     uint256 numerator;
     uint256 denominator;
 
-    if (activeReputationMiningCycle != colonyDecayRates[_colony].afterMiningCycle) {
+    uint256 chainId = _chainId;
```

Comment:
Could have done this as `uint256 chainId = isMiningChainId(_chainId) ? 0 : _chainId`